From 4c859eba57288eb17eb4603d67ef415b87503879 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Thu, 4 Aug 2022 12:15:46 +0200
Subject: [PATCH 1/1] [!99] replace `hostname` with `uname -n`o

The former belongs to package `inetutils`, which has some security
implications, so let's use the latter.
---
 client/scripts/linux | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/scripts/linux b/client/scripts/linux
index c4e51f6d..49eff40a 100755
--- a/client/scripts/linux
+++ b/client/scripts/linux
@@ -113,7 +113,7 @@ set_hostname() {
     local current_hostname
 
     if [ -n "$new_host_name" ]; then
-        current_hostname=$(hostname)
+        current_hostname=$(uname -n)
 
         # current host name is empty, '(none)' or 'localhost' or differs from new one from DHCP
         if [ -z "$current_hostname" ] ||
-- 
2.37.1

